### PR TITLE
3.5 - 13821 - Send To Location generates Movement Trail points.

### PR DESCRIPTION
**Pull request by BrentEaston**

### CODE REVIEW

File: vassal-app/src/main/java/VASSAL/counters/SendToLocation.java [hunk 1]
```diff
@@ -379,6 +380,12 @@ public Command myKeyEvent(KeyStroke stroke) {
         final ChangeTracker tracker = new ChangeTracker(this);
         setProperty(BACK_MAP, getMap());
         setProperty(BACK_POINT, getPosition());
+
+        // Mark moved and generate movement trail if compatibility pref turned on
+        if (Boolean.TRUE == GameModule.getGameModule().getPrefs().getValue(GlobalOptions.SEND_TO_LOCATION_MOVEMENT_TRAILS)) {
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/counters/SendToLocation.java:385]: Should use `.equals`, not `==` for these.

File: vassal-app/src/main/java/VASSAL/counters/SendToLocation.java [hunk 2]
```diff
@@ -402,6 +409,12 @@ else if (backCommand.matches(stroke)) {
       final ChangeTracker tracker = new ChangeTracker(this);
       setProperty(BACK_MAP, null);
       setProperty(BACK_POINT, null);
+
+      // Mark moved and generate movement trail if compatibility pref turned on
+      if (Boolean.TRUE == GameModule.getGameModule().getPrefs().getValue(GlobalOptions.SEND_TO_LOCATION_MOVEMENT_TRAILS)) {
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/counters/SendToLocation.java:414]: Ditto.

**Review by uckelman — CHANGES_REQUESTED** [review]

File: vassal-app/src/main/java/VASSAL/counters/SendToLocation.java (hunk 1 above)
Comment (BrentEaston) [reply vassal-app/src/main/java/VASSAL/counters/SendToLocation.java]: Oops. Fixed.

File: vassal-app/src/main/java/VASSAL/counters/SendToLocation.java (hunk 2 above)
Comment (BrentEaston) [reply vassal-app/src/main/java/VASSAL/counters/SendToLocation.java]: Fixed

**Review by uckelman — APPROVED** [review]

**Pull request merged**